index.html

```diff
@@ -1,26 +1,27 @@
 <!doctype html>
 <html>
   <head>
-    <link rel='stylesheet' href='styles.css'>
+    <title>John's page</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
   </head>
   <body>
     <header>
       <h1>The Header</h1>
     </header>
-    <aside>
+    <aside class="aside1">
```

Comment:
Got it, I thought it was the texts with no tag so I added a <p> tag around the text. I got it now.